packages/muelu/src/Interface/MueLu_ParameterListInterpreter_def.hpp

```diff
@@ -777,8 +791,7 @@ void ParameterListInterpreter<Scalar, LocalOrdinal, GlobalOrdinal, Node>::
       else if (preSmootherType == "RELAXATION")
         preSmootherParams = defaultSmootherParams;
 
-      if (preSmootherType == "CHEBYSHEV" && useMaxAbsDiagonalScaling)
-        preSmootherParams.set("chebyshev: use rowsumabs diagonal scaling", true);
+      setChebyshevSettings(preSmootherType, preSmootherParams);
```

Comment:
@MalachiTimothyPhillips Might be slightly tangential to this PR: could you check if this gets triggered if someone sets "smoother: type" = "chebyshev" or something else that's not all-caps? If not, could you add the same preprocessing as is done in MueLu_Ifpack2Smoother?